src/default-props.js

```diff
@@ -18,6 +18,21 @@ OverlayLoading.propTypes = {
   theme: PropTypes.any
 };
 
+const OverlayError = props => (
+  <div style={{ display: 'table', width: '100%', height: '100%', backgroundColor: fade(props.theme.palette.background.paper, 0.7) }}>
+    <div style={{ display: 'table-cell', width: '100%', height: '100%', verticalAlign: 'middle', textAlign: 'center' }}>
```

Comment:
nit: Be nice to abstract styling away from the html by having a js object that represent the style and pass it into this div.

Reply:
Thats the way mbrn decided tostyle the overlays. Where would you pass that style object for each div?